battenberg/cli.py

```diff
@@ -55,30 +56,40 @@ def main(ctx, o: str, verbose: bool):
 
 @main.command()
 @click.argument('template')
+@click.option(
+    '--initial-head',
```

Comment:
from `man git-init`, it looks like the long form for this option is named `--initial-branch`. should we be consistent?

```
       -b <branch-name, --initial-branch=<branch-name>
           Use the specified name for the initial branch in the newly created repository. If not specified, fall back to the default name: master.
```

Reply:
Looks like the terminology is inconsistent as `pygit2` calls it `initial-head`, happy to conform either way to be honest. https://www.pygit2.org/repository.html#pygit2.init_repository

Reply:
ah, thats unfortunate. i could go either way here, but i'd suggest being in line with git (cli) since developers are interfacing with that rather than pygit2 directly.

Reply:
Cool renamed `--initial-head` to `--initial-branch`